src/python/pants/backend/explorer/BUILD

```diff
@@ -28,8 +28,7 @@ __dependents_rules__(
     # have 3rd party dependencies that we don't want to leak to the core Pants distribution.
     (
         "*",
-        build_file_dir(),
-        build_file_dir() / "*",
+        build_file_dir() / "**",
```

Comment:
Will clean these up in a dedicated PR, where I also plan to break out the explorer specific 3rdparty deps so we can isolate them with visibility rules.